src/core/Jalasoft.TeamUp.Projects.Core/Validators/ProjectValidator.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Jalasoft.TeamUp.Projects.Core.Validators
+{
+    using System;
+    using FluentValidation;
+    using Jalasoft.TeamUp.Projects.Models;
+
+    public class ProjectValidator : AbstractValidator<Project>
+    {
+        public ProjectValidator()
+        {
+            this.RuleFor(project => project.Id)
+                .NotEmpty().NotNull();
+
+            this.RuleFor(project => project.Name)
+                .Length(3, 70)
+                .Matches("^[a-zñ A-ZÑ]+$")
+                .NotEmpty().NotNull();
+
+            this.RuleFor(project => project.Description)
+                .MaximumLength(160)
+                .WithMessage("It must not have more than 160 characters");
+
+            this.RuleFor(project => project.State)
+                .NotNull()
+                .Must(x => x == false || x == true);
```

Comment:
I think the proper way to do this is to change the value type for State to bool, if that is done this is not needed

Reply:
Yes, you are right it's already a Bool type, I deleted it because it is not necessary in this part